identity-issuance-api/src/commonMain/kotlin/com/android/identity/issuance/evidence/Openid4VciCredentialOffer.kt

```diff
@@ -0,0 +1,34 @@
+package com.android.identity.issuance.evidence
+
+import com.android.identity.cbor.annotation.CborSerializable
+
+@CborSerializable
+sealed class Openid4VciCredentialOffer(
+    val issuerUri: String,
+    val configurationId: String,
+    val authorizationServer: String?
+) {
+    companion object
+}
+
+class Openid4VciCredentialOfferPreauthorizedCode(
+    issuerUri: String,
+    configurationId: String,
+    authorizationServer: String?,
+    val preauthorizedCode: String,
+    val txCode: Openid4VciTxCode?
+) : Openid4VciCredentialOffer(issuerUri, configurationId, authorizationServer)
+
+class Openid4VciCredentialOfferAuthorizationCode(
+    issuerUri: String,
+    configurationId: String,
+    authorizationServer: String?,
+    val issuerState: String?
+) : Openid4VciCredentialOffer(issuerUri, configurationId, authorizationServer)
+
+@CborSerializable
+data class Openid4VciTxCode(
+    val message: String,
+    val numeric: Boolean,
```

Comment:
nit: Maybe rename to isNumeric? ...so it's obvious from the name that this is a booean...?

...If I hadn't seen that the type was Boolean, my initial guess would have been that this was a copy of "message," but as an integer rather than a string.

Reply:
Done